test/OpenTelemetry.Extensions.Docker.Tests/Resources/DockerResourceDetectorTests.cs

```diff
@@ -57,6 +57,13 @@ public class DockerResourceDetectorTests
     private const string CONTAINERID =
         "d86d75589bf6cc254f3e2cc29debdf85dde404998aa128997a819ff991827356";
 
+    // cgroupv2 line with container Id
+    private const string CGROUPLINEV2 =
```

Comment:
do you also need to check suffix or prefix for cgroup v2?
Also just a thought. Should we check prefix and suffix end to end, I mean with reading from file? If GetContainerId works with extracted cgroup string no matter V1 or V2, maybe we should have a separate test just for this function and do not mix it with reading the line from the file and parsing it depending on V1 or V2

Reply:
nevermind. I thought GetContainerId belongs to DockerResourceDetector(). Now I see it's from here. I'll cleanup my message later but my question is is still valid: do we need to check suffx and postfix for V2?

Reply:
No, it is not necessary